### PR TITLE
docs: Service Provider Tenant in the Mender UI

### PR DESCRIPTION
**Pull request by merlin-northern**
Ticket: MEN-7652

### CODE REVIEW

**Review by kjaskiewiczz — APPROVED** [review]
looks good, but I'd wait for @eysteins approval with merging

**Comment by merlin-northern** [comment]
> looks good, but I'd wait for @eysteins approval with merging

you just do not trust me at all, do you?  :*

File: 02.Overview/13.Multi-Tenancy/docs.md [hunk 1]
```diff
@@ -112,3 +112,37 @@ If you are using hosted Mender Enterprise please
 tenant setup using the Service Provider Tenant. For existing Tenants with a
 more complex configuration it may be treated as a
 [Consulting project](https://mender.io/pricing/mender-extras).
+
+In case of on-prem installations you can upgrade existing tenant to a service provider
```
[the remark this answers is not in the document]
Comment (estenberg) [reply 02.Overview/13.Multi-Tenancy/docs.md:116]: "For on-premise installations, you can promote an existing Tenant to a Service Provider Tenant with the following command:"

File: 02.Overview/13.Multi-Tenancy/docs.md [hunk 2]
```diff
@@ -112,3 +112,37 @@ If you are using hosted Mender Enterprise please
...
+tenantadm update-tenant --id TENANTID --service-provider
+```
+
+where `TENANTID` stands for the id of the tenant to update.
+
+## Service provider tenant in the Mender UI
+
+### Managing tenants
+
+After you log in to the tenant that is a service provider, you will see an empty tenants list:
```
[the remark this answers is not in the document]
Comment (estenberg) [reply 02.Overview/13.Multi-Tenancy/docs.md:129]: Service Provider

File: 02.Overview/13.Multi-Tenancy/docs.md [hunk 3]
```diff
@@ -112,3 +112,37 @@ If you are using hosted Mender Enterprise please
...
+
+## Service provider tenant in the Mender UI
+
+### Managing tenants
+
+After you log in to the tenant that is a service provider, you will see an empty tenants list: 
+
+[Service provider main view empty](sp0.png)
+
+Starting here you can add a new tenant:
```
Comment (estenberg) [inline 02.Overview/13.Multi-Tenancy/docs.md:133]: Tenant

File: 02.Overview/13.Multi-Tenancy/docs.md [hunk 4]
```diff
@@ -112,3 +112,37 @@ If you are using hosted Mender Enterprise please
...
+
+And it will appear in the list:
+
+[Service provider add](sp1.png)
+
+Now there are additional operations you can perform on it, by clicking the details link:
+
+[Service provider add](sp1-edit.png)
+
+As you can see, the Mender UI allows the service provider tenant to manage the children tenants.
```
Comment (estenberg) [inline 02.Overview/13.Multi-Tenancy/docs.md:145]: Service Provider... Tenants

File: 02.Overview/13.Multi-Tenancy/docs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (merlin-northern) [reply 02.Overview/13.Multi-Tenancy/docs.md]: fixed

File: 02.Overview/13.Multi-Tenancy/docs.md [hunk 5]
```diff
@@ -112,3 +112,37 @@ If you are using hosted Mender Enterprise please
 tenant setup using the Service Provider Tenant. For existing Tenants with a
 more complex configuration it may be treated as a
 [Consulting project](https://mender.io/pricing/mender-extras).
+
+In case of on-prem installations you can upgrade existing tenant to a service provider
+one with the following command line:
+
+```bash
+tenantadm update-tenant --id TENANTID --service-provider
```
[the remark this answers is not in the document]
Comment (merlin-northern) [reply 02.Overview/13.Multi-Tenancy/docs.md]: fixed

File: 02.Overview/13.Multi-Tenancy/docs.md [hunk 6]
```diff
@@ -112,3 +112,37 @@ If you are using hosted Mender Enterprise please
...
+```
+
+where `TENANTID` stands for the id of the tenant to update.
```
[the remark this answers is not in the document]
Comment (merlin-northern) [reply 02.Overview/13.Multi-Tenancy/docs.md]: fixed

File: 02.Overview/13.Multi-Tenancy/docs.md [hunk 7]
```diff
@@ -112,3 +112,37 @@ If you are using hosted Mender Enterprise please
...
+
+## Service provider tenant in the Mender UI
```
[the remark this answers is not in the document]
Comment (merlin-northern) [reply 02.Overview/13.Multi-Tenancy/docs.md]: fixed

File: 02.Overview/13.Multi-Tenancy/docs.md [hunk 8]
```diff
@@ -112,3 +112,37 @@ If you are using hosted Mender Enterprise please
...
+
+### Managing tenants
```
[the remark this answers is not in the document]
Comment (merlin-northern) [reply 02.Overview/13.Multi-Tenancy/docs.md]: fixed.

File: 02.Overview/13.Multi-Tenancy/docs.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (merlin-northern) [reply 02.Overview/13.Multi-Tenancy/docs.md]: fixed

File: 02.Overview/13.Multi-Tenancy/docs.md (hunk 4 above)
Comment (merlin-northern) [reply 02.Overview/13.Multi-Tenancy/docs.md]: fixed

File: 02.Overview/13.Multi-Tenancy/docs.md [hunk 9]
```diff
@@ -112,3 +112,39 @@ If you are using hosted Mender Enterprise please
...
+
+For on-premise installations, you can promote an existing Tenant to a Service Provider Tenant with the following command:
+
+```bash
+TENANTID=your-tenant-id-here
+tenantadm update-tenant --id "${TENANTID}" --service-provider
+```
+
+where `TENANTID` variable holds the ID of the Tenant to promote.
+Please note, you have to execute it from the inside of the `tenantadm`
```
Comment (estenberg) [inline 02.Overview/13.Multi-Tenancy/docs.md:124]: Execute what? The command above, or a different command?

One other thing, how do they get hold of the Tenant ID? Can you describe that before the command? They always will need the Tenant ID to run this command, and if we don't describe how to get it they can't proceed.

**Review by estenberg — CHANGES_REQUESTED** [review]
One small thing, re obtaining Tenant ID, at least some guidance needed I think.

File: 02.Overview/13.Multi-Tenancy/docs.md [hunk 10]
```diff
@@ -112,3 +112,52 @@ If you are using hosted Mender Enterprise please
...
+For on-premise installations, you can promote an existing Tenant to a Service Provider Tenant with the following command:
+
+```bash
+TENANTID=your-tenant-id-here
+tenantadm update-tenant --id "${TENANTID}" --service-provider
+```
+
+where `TENANTID` variable holds the ID of the Tenant to promote.
+Please note, you have to execute the above command from the inside of the `tenantadm`
+container. For the on-prem installations you can find the Tenant id, by listing the Tenants
```
Comment (estenberg) [inline 02.Overview/13.Multi-Tenancy/docs.md:125]: For on-premise installations you can find the Tenant ID by listing the Tenants using the following command:

**Review by estenberg — CHANGES_REQUESTED** [review]

File: 02.Overview/13.Multi-Tenancy/docs.md (hunk 10 above)
Comment (merlin-northern) [reply 02.Overview/13.Multi-Tenancy/docs.md]: fixed.

**Review by estenberg — APPROVED** [review]

**Pull request merged**